PWGEM/PhotonMeson/TableProducer/createPCM.cxx

```diff
@@ -189,29 +191,29 @@ struct createPCM {
     }
   }
 
-  float v0_alpha(float pxpos, float pypos, float pzpos, float pxneg, float pyneg, float pzneg)
+  static float v0_alpha(float pxpos, float pypos, float pzpos, float pxneg, float pyneg, float pzneg)
   {
     float momTot = RecoDecay::p(pxpos + pxneg, pypos + pyneg, pzpos + pzneg);
-    float lQlNeg = RecoDecay::dotProd(array{pxneg, pyneg, pzneg}, array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg}) / momTot;
-    float lQlPos = RecoDecay::dotProd(array{pxpos, pypos, pzpos}, array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg}) / momTot;
+    float lQlNeg = RecoDecay::dotProd(std::array{pxneg, pyneg, pzneg}, std::array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg}) / momTot;
+    float lQlPos = RecoDecay::dotProd(std::array{pxpos, pypos, pzpos}, std::array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg}) / momTot;
     return (lQlPos - lQlNeg) / (lQlPos + lQlNeg);
   }
-  float v0_qt(float pxpos, float pypos, float pzpos, float pxneg, float pyneg, float pzneg)
+  static float v0_qt(float pxpos, float pypos, float pzpos, float pxneg, float pyneg, float pzneg)
   {
     float momTot = RecoDecay::p2(pxpos + pxneg, pypos + pyneg, pzpos + pzneg);
-    float dp = RecoDecay::dotProd(array{pxneg, pyneg, pzneg}, array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg});
+    float dp = RecoDecay::dotProd(std::array{pxneg, pyneg, pzneg}, std::array{pxpos + pxneg, pypos + pyneg, pzpos + pzneg});
     return std::sqrt(RecoDecay::p2(pxneg, pyneg, pzneg) - dp * dp / momTot);
   }
 
   template <typename TTrack>
   bool reconstructV0(TTrack const& ele, TTrack const& pos)
   {
-    bool isITSonly_pos = pos.hasITS() & !pos.hasTPC();
-    bool isITSonly_ele = ele.hasITS() & !ele.hasTPC();
-    bool isTPConly_pos = !pos.hasITS() & pos.hasTPC();
```

Comment:
isTPConly here means that tracks that do not have ITS hits. Namely, TPConly, TPC-TOF, TPC-TRD, TPC-TRD-TOF should be accepted. I am sorry for confusion.

Reply:
Thank you for clarifying.